plugins/todoist/todoist.go

```diff
@@ -0,0 +1,25 @@
+package todoist
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func TodoistCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Todoist CLI",
+		Runs:    []string{"todoist"},
+		DocsURL: sdk.URL("https://github.com/sachaos/todoist"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
```

Comment:
I think we should remove this - as it stands, if you run `todoist` without any arguments it just asks for your API token and doesn't give you the help text with the list of commands. What do you think?